src/models/UserMapper.php

```diff
@@ -38,6 +38,7 @@ public function getVerboseFields()
             "username"         => "username",
             "full_name"        => "full_name",
             "twitter_username" => "twitter_username",
+            "trusted"          => "trusted"
```

Comment:
I think it's more than just the user itself - any admin might need to be able to check the trusted state potentially.

I guess a nicer way to do it would be to only have the trusted field if the user is a site admin or the user, however AFAIK we don't have a clean way of defining mapper fields dependent on role? As such, I don't think exposing "trusted" is an issue from a security POV, so perhaps the bigger case is that we need to have a way of defining "admin" fields which are only returned whan the authenticated user has admin access to the current object?